packages/runner-ct/src/lib/mobx.ts

```diff
@@ -0,0 +1,12 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+
+/**
+ * Wraps MobX `observer` to properly add a component `displayName` for debugging purposes
+ */
+// Using `any` as a default generic here has the interesting property of allowing types to properly passthrough, as otherwise the prop type would default to `unknown`
+export const namedObserver = <TComponent extends React.ComponentType<TProps>, TProps = any>(displayName: string, component: TComponent): TComponent => {
```

Comment:
Fine with `T` and `I` prefixes, but if we are going with this convention we likely want to change the entire code-base?

Personally I don't see a lot of value in `T` and `I` prefixes, though.

Reply:
You guessed it. I was also in the `I` camp until Tim made me stop.

I'm fine with removing it, though I personally really like indicating generics with `T` so that they don't collide with real types, and to explicitly call out to the programmer that this is a generic.